Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml

```diff
@@ -136,7 +136,7 @@ script:
     - contextPath: CybleIntel.collection.names
       description: Available collection names for the feed service.
     description: Get the data feed collection names for the taxii feed.
-  dockerimage: demisto/taxii-server:1.0.0.84385
+  dockerimage: demisto/taxii-server:1.0.0.85407
```

Comment:
Make sure to comment this update on the release notes.

Reply:
Done